src/lib/components/ProjectsPanel/components/ProjectsPanel.module.css

```diff
@@ -0,0 +1,100 @@
+.toggle {
+  display: inline-flex;
+  padding: 8px;
+  transition: 0.2s all ease;
```

Comment:
It seems that we can try 150ms, especially on smaller screens. However, this is a taste :)

Reply:
50ms is too small difference)